app/models/condition.rb

```diff
@@ -87,7 +87,10 @@ def self.evaluate(cond, rec, inputs={}, attr=:expression)
   def self.do_eval(expr)
     # Remove this [DEPRECATION] in the next version
     if expr =~ /^__start_ruby__\s?__start_context__\s?(.*)\s?__type__\s?(.*)\s?__end_context__\s?__start_script__\s?(.*)\s?__end_script__\s?__end_ruby__$/im
-      raise "Ruby scripts in expressions are no longer supported. Please use the regular expression feature of conditions instead."
+      context, col_type, script = [$1, $2, $3]
+      context = MiqExpression.quote(context, col_type)
+      result = SafeNamespace.eval_script(script, context)
```

Comment:
What example scripts do we support here?  These are stored in the MiqExpression somewhere right?  I'm wondering if this feature is even used and if so, if we can migrate these to something that doesn't need to do eval at all.